flask_opentracing/tracing.py

```diff
@@ -87,17 +88,26 @@ def _before_request_fn(self, attributes):
             span = self._tracer.start_span(operation_name=operation_name)
 
         self._current_spans[request] = span
+
+        span.set_tag(tags.COMPONENT, 'Flask')
+        span.set_tag(tags.HTTP_METHOD, request.method)
+        span.set_tag(tags.HTTP_URL, request.base_url)
+        span.set_tag(tags.SPAN_KIND, tags.SPAN_KIND_RPC_SERVER)
+
         for attr in attributes:
             if hasattr(request, attr):
                 payload = str(getattr(request, attr))
                 if payload:
                     span.set_tag(attr, payload)
 
-    def _after_request_fn(self):
+    def _after_request_fn(self, response=None):
```

Comment:
why is the new param optional? Is someone calling it other than L30?

Reply:
Yes, our actual wrapper is using it, when instead of instrumenting it all, you have 

```python
@tracing.trace()
def my_request_handler():
   ...
```

In that case, the response is not automatically retrieved upon method decoration (after a few initial tries, I failed to see an easy, correct way to do that - will fix that in another PR).